app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/demographics/LabOrderSender.java

```diff
@@ -1,6 +1,12 @@
 package gov.hhs.cdc.trustedintermediary.etor.demographics;
 
+import gov.hhs.cdc.trustedintermediary.wrappers.SecretRetrievalException;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
 /** Interface for sending a lab order. */
 public interface LabOrderSender {
-    void sendOrder(LabOrder<?> order) throws UnableToSendLabOrderException;
+    void sendOrder(LabOrder<?> order)
+            throws UnableToSendLabOrderException, SecretRetrievalException, InvalidKeySpecException,
+                    NoSuchAlgorithmException;
```

Comment:
Same thing as the above idea with the exception.

Reply:
Same as above